src/sentry/api/endpoints/organization_release_meta.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import absolute_import
+
+from collections import defaultdict
+
+from rest_framework.response import Response
+
+from sentry.api.base import DocSection
+from sentry.api.bases.organization import OrganizationReleasesBaseEndpoint
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.models import (
+    Release,
+    CommitFileChange,
+    ReleaseFile,
+    ReleaseCommit,
+    ReleaseProject,
+    ProjectPlatform,
+)
+from sentry.api.serializers.models.release import expose_version_info
+
+
+class OrganizationReleaseMetaEndpoint(OrganizationReleasesBaseEndpoint):
+    doc_section = DocSection.RELEASES
+
+    def get(self, request, organization, version):
+        """
+        Retrieve an Organization's Release's Associated Meta Data
+        `````````````````````````````````````````````````````````
+
+        The data returned from here is auxiliary meta data that the UI uses.
+
+        :pparam string organization_slug: the slug of the organization the
+                                          release belongs to.
+        :pparam string version: the version identifier of the release.
+        :auth: required
+        """
+        try:
+            release = Release.objects.get(organization_id=organization.id, version=version)
+        except Release.DoesNotExist:
+            raise ResourceDoesNotExist
+
+        if not self.has_release_permission(request, organization, release):
+            raise ResourceDoesNotExist
+
+        commit_files_changed = CommitFileChange.objects.filter(
+            commit_id__in=ReleaseCommit.objects.filter(release=release).values_list(
+                "commit_id", flat=True
+            )
+        ).count()
```

Comment:
Should we count only CommitFileChange with unique filename?

Reply:
This is the same logic we use in the other part